go/api/commands.go

```diff
@@ -704,6 +704,28 @@ type HashCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/hstrlen/
 	HStrLen(key string, field string) (Result[int64], error)
+
+	// Increments the number stored at `field` in the hash stored at `key` by increment.
+	// By using a negative increment value, the value stored at `field` in the hash stored at `key` is decremented.
+	// If `field` or `key` does not exist, it is set to 0 before performing the operation.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	// 	key - The key of the hash.
+	// 	field - The field in the hash stored at `key` to increment its value.
+	// 	increment - The amount to increment.
+	//
+	// Return value:
+	// 	The Result[int64] value of `field` in the hash stored at `key` after the increment.
+	//
+	// For example:
+	//  hsetResult, err := client.HSet("key", map[string]string{"field": "10"})
```

Comment:
```suggestion
	//  _, err := client.HSet("key", map[string]string{"field": "10"})
```
maybe, since we aren't using this result?

Reply:
This is more clear. 
Unless, there is some language specific reason, I would prefer clarity on what is returned.

Reply:
I think our examples should be executable code so having unused variables declared and assigned feels a bit off to me. I also don't believe there's less clarity if you omit this since hset isn't the command we are trying to show here. Then again if you believe it is worth including I don't have a strong opinion against it and you can merge as is.

Reply:
Sorry. You are right TJ. The comments had split the code so I didn't see the rest of the example.  The hsetResult should be replaced with `_` since the return value is unused.

Reply:
Updated to _.